threedi_api_qgis_client/utils.py

```diff
@@ -119,3 +119,19 @@ def check_download_checksum(download, filename):
         data = file_to_check.read()
         md5_returned = hashlib.md5(data).hexdigest()
         return download.etag == md5_returned
+
+
+def handle_api_exceptions(e):
```

Comment:
I'd prefer `extract_error_message()` or something like that.
"Handle api exception", then I'd expect it to be *handled* for me, not for it to return me the error message.

Reply:
Done.